GUI/src/cz/fidentis/gui/comparison_two_faces/Bundle.properties

```diff
@@ -64,8 +63,6 @@ PairRegistrationConfiguration.jTable1.columnModel.title3_1=Title 4
 =======
 PairComparisonResults.jButton10.text=Export registered faces...
 >>>>>>> .r154
```

Comment:
toto sice nesuvisi s tvojimi zmenami ale vypada ze niekto niekedy davno commitol merge conflict... cudujem sa ze to funguje vobec :D